its/src/test/java/com/sonar/it/vbnet/AutoGeneratedTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * SonarSource :: C# :: ITs :: Plugin
+ * Copyright (C) 2011-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.it.vbnet;
+
+import com.sonar.it.shared.TestUtils;
+import com.sonar.orchestrator.Orchestrator;
+import com.sonar.orchestrator.http.HttpMethod;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static com.sonar.it.vbnet.Tests.getMeasureAsInt;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AutoGeneratedTest {
+    @ClassRule
+    public static final Orchestrator orchestrator = Tests.ORCHESTRATOR;
+
+    @Rule
+    public TemporaryFolder temp = new TemporaryFolder();
+
+    @Before
+    public void init() {
+      orchestrator.resetData();
+    }
+
+    @Test
+    public void autogenerated_code_with_analyzeGenerated_false() throws Exception {
+      final String PROJECT_NAME = "VbAutoGeneratedFiles";
```

Comment:
`PROJECT_NAME` could be a static final field as the same value is used for the other test method